src/routes/tickets/controllers/ticketController.ts

```diff
@@ -2,98 +2,127 @@ import { RequestHandler } from 'express';
 import { body } from 'express-validator';
 
 export enum TicketStatus {
-	open = 1,
-	updated,
-	snoozed,
-	closed,
+  open = 1,
+  updated,
+  snoozed,
+  closed,
 }
 
+export interface JSONContent {
+  type?: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  attrs?: Record<string, any>;
+  content?: JSONContent[];
+  marks?: {
+    type: string;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    attrs?: Record<string, any>;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    [key: string]: any;
+  }[];
+  text?: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  [key: string]: any;
+}
```

Comment:
What is this for? Comments pls <3

Reply:
Oh yeah only commented this on the FE I think. This comes from the lib tiptap JSONContent is essentially the type that is returned from editor.getJSON() on the FE.

Reply:
`TipTapContent` -> Maybe as a better name?

Reply:
Maybe, but the type on the Fe is JSONContent (This type/interface is taken from their lib) so I think it makes more sense to keep it the same between the two projects/services

Reply:
@binaryoverload

Reply:
I disagree personally, I would name it what it makes sense to. Even when matching with the FE, a different name would offer much more clarity.

If that's still a no, then please at least add a comment.

@mrjvs opinion?

Reply:
agree with bin bin

Reply:
Alright, the council has spoken. 👨‍⚖️